main.py

```diff
@@ -0,0 +1,191 @@
+from typing import Optional
+import requests
+from bs4 import BeautifulSoup
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.sql import exists
+import db_book
+import db_other
+
+
+def search_book_info_list() -> list:
+    return ["Sku", "ISBN 13", "ISBN 10", "Title", "Author", "Publisher", "Year published"]
+
+
+def search_for_other_than_book() -> list:
+    return ["Sku", "Title", "Studio", "EAN", "Release date"]
+
+
+def get_response_of_url(url: str) -> Optional[str]:
```

Comment:
Maybe 
get_response_text_of_url

just to indicate you're going to be returning the text, and not the full response
=